usage-based-subscriptions/server/dotnet/Controllers/ReportUsage.cs

```diff
@@ -0,0 +1,76 @@
+using System;
```

Comment:
Lines 1-35 I've added but I have no idea if they're right. Lines 37+ were copy and pasted.